src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  for (const keyToInverse in robot) {
+    if (inversedRobot.hasOwnProperty(robot[keyToInverse])) {
```

Comment:
It is better to create a variable for robot[keyToInverse] since it is used more than one time.

Reply:
Good idea. I've implemented it.